tests/random_attack.py

```diff
@@ -757,14 +806,16 @@ def dedup_names(self):
                                str(args.seed) + ".py")
         python_output = open(outfile, "wt")
         python_output.write("#!/usr/bin/env python\n")
+        python_output.write("import sys; sys.path += ('.', '..')\n")
         python_output.write("import datatable as dt\n")
         python_output.write("import numpy as np\n")
         python_output.write("from datatable import f\n")
-        python_output.write("from datatable.internal import frame_integrity_check\n\n")
+        python_output.write("from datatable.internal import frame_integrity_check\n")
```

Comment:
The extra `\n` was needed to visually separate imports from the rest of the code.

Reply:
Yeah, I was cleaning my debug code and this one got affected, will revert it back.